packages/date-picker/src/_subcomponents/FloatingCalendarWrapper/FloatingCalendarWrapper.module.scss

```diff
@@ -1,9 +1,11 @@
 @import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/spacing";
 
 .calendarWrapper {
   background-color: $color-white;
   z-index: 1010;
```

Comment:
This `z-index` appears arbitrary. Would this cause concern when laid out with other components?